tests/test_api.py

```diff
@@ -208,28 +209,30 @@ def setUp(self):
         self.tag = models.Tag.objects.create(label="Test", color="#FF5733")
 
     def test_create_new_tag(self):
-        data = {"label": self.tag.label, "color": self.tag.color}
+        self.assertEqual(models.Tag.objects.count(), 1)
+        data = {"label": "testLabel", "color": "#91EE28"}
         response = self.client.post("/api/tags", data)
-        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.status_code, 201)
+        self.assertEqual(models.Tag.objects.count(), 2)
 
     def test_list_all_tags(self):
         response = self.client.get("/api/tags")
         self.assertEqual(response.status_code, 200)
 
     def test_get_tag_by_id(self):
-        tag_id = os.environ.get("TEST_TAG_ID", 1)
+        tag_id = os.environ.get("TEST_TAG_ID", "1")
```

Comment:
better to move this line to `setUp` function like `self.tag_id = os.environ.get("TEST_TAG_ID", "1")`

Reply:
so to remove duplication from L228 and L234 too.